### PR TITLE
i18n(pt-BR): Translate tutorial `pages-2` first half 

### PR DESCRIPTION
**Pull request by Yan-Thomas**
#### What kind of changes does this PR include?

- Translated content

#### Description

PT-BR translation for the first half of `pages-2`.

### CODE REVIEW

File: src/content/docs/pt-br/tutorial/2-pages/1.mdx [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+  <p>Este site será atualizado enquanto eu completo mais desse tutorial, então continue verificando e veja como minha jornada está indo!</p>
+</body>
+```
+
+Agora, visite sua página `/sobre` na aba do seu navegador novamente, e você deve ver seu conteúdo atualizado.
+
+## Adicione links de navegação
+
+Para fazer mais fácil de pré-visualizar todas as suas páginas, adicione links de navegação de página HTML antes do seu `<h1>` noa topo de ambas as suas páginas (`index.astro` e `sobre.astro`):
```
Comment (lumamontes) [inline src/content/docs/pt-br/tutorial/2-pages/1.mdx:61]:
```suggestion
Para ser mais fácil de pré-visualizar todas as suas páginas, adicione links de navegação de página HTML antes do seu `<h1>` no topo de ambas as suas páginas (`index.astro` e `sobre.astro`):
```

File: src/content/docs/pt-br/tutorial/2-pages/1.mdx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+<a href="/sobre/">Sobre</a>
+
+<h1>Sobre Mim</h1>
+<h2>... e meu novo site Astro!</h2>
+```
+
+Verifique que você pode clicar nesses links para ir e voltar entre as páginas no seu site.
+
+:::note
+Diferente de vários frameworks, Astro utiliza elementos `<a>` padrões do HTML para navegar entre páginas (também chamadas rotas_), com recarregamento tradicional de página.
```
Comment (lumamontes) [inline src/content/docs/pt-br/tutorial/2-pages/1.mdx:74]:
```suggestion
Diferente de vários frameworks, Astro utiliza elementos `<a>` padrões do HTML para navegar entre páginas (também chamadas rotas), com recarregamento tradicional de página.
```

File: src/content/docs/pt-br/tutorial/2-pages/2.mdx [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+
+    Bem-vindo ao meu _novo blog_ sobre aprendendo Astro! Aqui, eu irei compartilhar minha jornada de aprendizado enquanto eu construo um novo website.
+
+    ## O que eu realizei
+
+    1. **Instalando Astro**: Primeiro, eu criei um novo projeto Astro e configurei minhas contas online.
+
+    2. **Fazendo Páginas**: E então eu aprendi como fazer páginas criando novos arquivos `.astro` e os colocando na pasta `src/pages/`.
+
+    3. **Fazendo Postagens no Blog**: Esta é minha primeira postagem no blog! Eu agroa tenho páginas Astro e postagens em Markdown!
```
Comment (lumamontes) [inline src/content/docs/pt-br/tutorial/2-pages/2.mdx:65]:
```suggestion
    3. **Fazendo Postagens no Blog**: Esta é minha primeira postagem no blog! Eu agora tenho páginas Astro e postagens em Markdown!
```

File: src/content/docs/pt-br/tutorial/2-pages/index.mdx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+import Box from '~/components/tutorial/Box.astro';
+
+Agora que você tem um site funcional na web, vamos adicionar páginas e postagens!
+
+## Onde estamos agora?
+
+Você pode encontrar o código para este ponto no tutorial no [GitHub](https://github.com/withastro/blog-tutorial-demo/tree/unit-2/start) ou
+[StackBlitz](https://stackblitz.com/github/withastro/blog-tutorial-demo/tree/unit-2/start?file=src/pages/index.astro).
+
+Verifique seu código contra esse exemplo, ou se você acabou de se juntar a nós, faça um fork no StackBlitz para começar a codificar junto do seu navegador a partir de agora.
```
Comment (lumamontes) [inline src/content/docs/pt-br/tutorial/2-pages/index.mdx:21]:
```suggestion
Compare seu código contra esse exemplo, ou se você acabou de se juntar a nós, faça um fork no StackBlitz para começar a codificar junto do seu navegador a partir de agora.
```

**Review by lumamontes — COMMENTED** [review]
Great work ✨  i left some suggestions on this one, mostly for typos! besides this, i think it's all good

**Comment by Yan-Thomas** [comment]
Thanks @lumamontes, appreciate the review! 🙌

**Pull request merged**